docs/src/main/paradox/docs/fusion/plugins.md

```diff
@@ -1,22 +1,25 @@
 # Plugins
-A resource returned by the Nexus Delta API is usually a JSON-LD document. By default, Nexus Fusion displays JSON-LD in a 
+
+A resource returned by the Nexus Delta API is usually a JSON-LD document. By default, Nexus Fusion displays JSON-LD in a
 code editor. If the user has edit access to the document, they can update the resource in the editor.
```

Comment:
Nice to have: Personally, I prefer "edit rights" to "edit access". Personal opinion though, so feel free to ignore.